config/helm/chart/default/values.yaml

```
@@ -194,3 +194,14 @@ csidriver:
         cpu: 20m
         memory: 30Mi
 
+rbac:
+  activeGate:
+    annotations:
```

```suggestion
    annotations: {}
```
can we set an explicit empty value so it doesn't look so forgotten?

---

Done